drivers/bridge/bridge.go

```diff
@@ -558,11 +566,12 @@ func (d *driver) createNetwork(config *networkConfiguration) error {
 
 	// Create and set network handler in driver
 	network := &bridgeNetwork{
-		id:         config.ID,
-		endpoints:  make(map[string]*bridgeEndpoint),
-		config:     config,
-		portMapper: portmapper.New(),
-		driver:     d,
+		id:            config.ID,
+		endpoints:     make(map[string]*bridgeEndpoint),
+		config:        config,
+		portMapper:    portmapper.New(),
+		driver:        d,
+		iptCleanFuncs: make(iptablesCleanFuncs, 0),
```

Comment:
I think you do not need to initialize `iptCleanFuncs`. You are using `append` to populate it and `range` when iterating.
Both handle the `nil` collection.